core/api/storage_miner/storage_api.hpp

```diff
@@ -115,6 +122,53 @@ namespace fc::api {
                SectorSize,
                bool)
 
+    API_METHOD(ReturnAddPiece,
+               void,
+               const CallId &,
+               const boost::optional<PieceInfo> &,
+               const boost::optional<CallError> &)
+    API_METHOD(ReturnSealPreCommit1,
+               void,
+               const CallId &,
+               const boost::optional<PreCommit1Output> &,
```

Comment:
in lotus result value is not optional (go struct can't be nil).

Reply:
Fixed, change to default